### ISSUE TITLE
CMS VAMC Facility View screen nested fieldsets have no associated legend

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10129]
## Description
**Still an issue in D10**

In the CMS, the View screen for the VAMC Facility content type has nested fields displayed using `<fieldset>` tags but without using an associated `<legend>` as well.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183902958-a3bc1a9f-6d8c-406c-b338-d24eca9b99e8.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Determine how the structure should be remediated (add a legend, use the current heading as the legend, or fieldset removed)
- [ ] Implement the recommended structure

## Definition of done
- [x] Code completed; PR created
- [x] Review instance created/available; internal team notified to start reviews
- [x] Automated tests passed 
- [x] Code review (meets established quality and security standards)
- [x] PR approved
- [ ] Design review (meets design & UX requirements)
- [x] A11y review (meets accessibility standards)
- [x] Functional/feature review (meets acceptance criteria)
- [x] Testing completed; dev team notified to deploy
- [ ] PO/PM/external verify in prod
- [ ] Documentation updated

**Comment by r-bartlett-gsa** [comment]
@Becapa How should this be tested on prod?
